demos/ids-checkbox-group/example.html

```diff
@@ -0,0 +1,9 @@
+<ids-layout-grid auto="true">
```

Comment:
```suggestion
<ids-layout-grid auto="true">
  <ids-layout-grid-cell>
    <ids-text font-size="12" type="h1">Checkbox Groups</ids-text>
  </ids-layout-grid-cell>
</ids-layout-grid>
<ids-layout-grid auto="true">
```

Adding this label above should fix the error in the tests @jeffwa428